docs/docs/components/quickstart/nodejs/actions/README.md

```diff
@@ -0,0 +1,455 @@
+
+```javascript
+const axios = require("axios")
+
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.2",
+  type: "action",
+  props: {
+    name: {
+      type: "string",
+      label: "Name",
+    }
+  },
+  async run() {
+    return `hello ${this.name}!`
+  },
+}
+```
+
+> **Note:** To use most npm packages on Pipedream, just require them — there is no `package.json` or `npm install` required.
+
+Then, update the `run()` method to:
+
+- Make a request to the following endpoint for the Star Wars API: `https://swapi.dev/api/people/1/`
+- Reference the `name` field of the payload returned by the API
+
+```javascript
+const axios = require("axios")
+
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.2",
+  type: "action",
+  props: {
+    name: {
+      type: "string",
+      label: "Name",
+    }
+  },
+  async run() {
+    const response = await axios.get("https://swapi.dev/api/people/1/")
+		return `hello ${response.data.name}!`
+  },
+}
+```
+
+Next, remove the `name` prop since we're no longer using it.
+
+```javascript
+const axios = require("axios")
+
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.2",
+  type: "action",
+  props: {},
+  async run() {
+    const response = await axios.get("https://swapi.dev/api/people/1/")
+		return `hello ${response.data.name}!`
+  },
+}
+```
+
+Finally, update the version to `0.0.3`. If you fail to update the version, the CLI will throw an error.
+
+```javascript
+const axios = require("axios")
+
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.3",
+  type: "action",
+  props: {},
+  async run() {
+    const response = await axios.get("https://swapi.dev/api/people/1/")
+		return `hello ${response.data.name}!`
+  },
+}
+```
+
+Save the file and run the `pd publish` command again to update the action in your account.
+
+```
+pd publish action.js
+```
+
+The CLI will update the component in your account with the key `action_demo`. You should see something like this:
+
+```
+sc_ZriKEn  Action Demo                             0.0.3    1 second ago         action_demo
+```
+
+Follow the steps in the previous example to update and run the action in your workflow. You should see `hello Luke Skywalker!` as the return value for the step.
+
+### Use Managed Auth
+
+For the last example, we'll use Pipedream managed auth to retrieve and emit data from the Github API (which uses OAuth for authentication). First, remove the line that requires `axios` and clear the `run()` function from the last example. Your code should look like this:
+
+```javascript
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.3",
+  type: "action",
+  async run() {
```

Comment:
Really small nit but most editors will collapse empty functions to 

```javascript
async run() { }
```

so that might save you a couple of lines.